spec/models/dimensions/item_spec.rb

```diff
@@ -10,11 +10,37 @@
     end
 
     context "when valid schema" do
-      it "returns content json if details.body exists" do
-        json = build_raw_json(body: 'the-body')
-
-        item = create(:dimensions_item, raw_json: json)
-        expect(item.get_content).to eq('the-body')
+      it 'returns content json from :body for all valid formats' do
```

Comment:
This seems like a unit test for the parser. If we unit test the method, it would be enough with stubbing the response. I would also add an integration [test as described in Mat's comment](https://github.com/alphagov/content-performance-manager/pull/521#discussion_r171204318).

Reply:
Addressed in https://github.com/alphagov/content-performance-manager/pull/576